scripts/releases-json.py

```diff
@@ -0,0 +1,205 @@
+"""
+Generate a releases.json entry for a new firmware version.
+
+Expects that the binaries have the correct names and are placed in the correct directories,
+incl. translation blobs.
+
+Usage:
+
+    cd scripts
+    poetry install
+    poetry run python releases-json.py 2.8.1 --changelog=changelog.txt
+
+You need `prettier` on your system. For example: `nix-shell -p nodePackages.prettier`.
+
+This will scan all models for a newly added version 2.8.1 and generate the corresponding
+entries. If a bootloader is updated, you still have to edit releases.json manually,
+or provide the new bootloader version with the --bootloader-version option.
+
+MISSING FEATURES:
+* T1B1 is not supported. It should be enough to add an entry to BOOTLOADER_MIN
+  and FIRMWARE_MIN, maybe.
+* Can't do a bitcoin-only specific changelog. Not sure how that would look in the notion
+  format. Someone will need to fix that.
+
+The changelog format is as copypasted from notion. Expected format:
+
+------------------
+### T3T1 **[2.8.1] (21th August 2024)**
+
+**Added**
+
+- Added PIN keyboard animation. [#3885](https://github.com/trezor/trezor-firmware/pull/3885)
+- Added menu entry animation. [#3896](https://github.com/trezor/trezor-firmware/pull/3896)
+- Added screen brightness settings. [#3969](https://github.com/trezor/trezor-firmware/pull/3969)
+
+**Fixed**
+
+- Fixed title sometimes not fitting into result screen. [#4023](https://github.com/trezor/trezor-firmware/pull/4023)
+
+### T2T1 **[2.8.1] (21th August 2024)**
+
+**Fixed**
+
+- Fixed displaying of a progress indicator for the formatting operation. [#3035](https://github.com/trezor/trezor-firmware/pull/3035)
+------------------
+"""
+
+from __future__ import annotations
+
+import json
+import subprocess
+import re
+import typing as t
+from pathlib import Path
+
+import click
+from trezorlib import firmware
+
+ROOT = Path(__file__).parent.parent.resolve()
```

Comment:
The URLs should start with `data/firmware/`, but they start with `firmware/`

Reply:
fixed